zulip_botserver/zulip_botserver/server.py

```diff
@@ -33,18 +33,29 @@ def load_lib_modules():
     # type: () -> None
     for bot in available_bots:
         try:
-            module_name = 'zulip_bots.{bot}.{bot}'.format(bot=bot)
-            bots_lib_module[bot] = import_module(module_name)
+            module_name = 'zulip_bots.bots.{bot}.{bot}'.format(bot=bot)
+            lib_module = import_module(module_name)
+            bots_lib_module[bot] = lib_module
         except ImportError:
             print("\n Import Error: Bot \"{}\" doesn't exists. Please make sure you have set up the flaskbotrc "
                   "file correctly.\n".format(bot))
-            sys.exit(1)
+
+def get_bot_lib_module(bot):
+    # type: (str) -> Any
+    if bot in bots_lib_module.keys():
+        return bots_lib_module[bot]
+    return None
 
 app = Flask(__name__)
 
 @app.route('/bots/<bot>', methods=['POST'])
 def handle_bot(bot):
     # type: (str) -> Union[str, BadRequest]
+    lib_module = get_bot_lib_module(bot)
+    if lib_module is None:
+        return BadRequest("Either the configuration for bot {} is missing or the Bot Handler Class cannot be "
+                          "fetched.".format(bot))
+
```

Comment:
@eeshangarg Thanks for reviewing this. There exists a similar test case for this, I will refactor it. :)

The main motivation for this was to fix the case where there exists a bot which works purely as an outgoing webhook bot i.e. posts data to a third party URL but doesn't have the bot code. Hence the bot handler couldn't be fetched and zulip-bot-server would have failed. Hence I added the check and I tried to convey the possible info through the message. But I like your idea, and I think error message such as above would convey the most info about the error. 
`BadRequest("Can't find the configuration or bot code for bot {}. Make sure that the `zulip_bots` package is installed!".format(bot)))`  
What do you think?